bindings/rascal/models/IP_generic_md.py

```diff
@@ -0,0 +1,103 @@
+import ase.io
+import ase.units
+import numpy as np
+
+from ..utils import BaseIO, load_obj
+from ..neighbourlist.structure_manager import AtomsList, unpack_ase
+
+
+class GenericMDCalculator(BaseIO):
+
+    """Generic MD driver for a librascal model
+
+    Initialize with model JSON and a structure template, and calculate
+    energies and forces based on position/cell updates _assuming the
+    order and identity of atoms does not change_.
```

Comment:
Is this assumption checked? Checking that we have a constant number of atoms should be cheap enough, and even the atomic identities should be fast to check.

Reply:
No, but almost any MD code operates under this same assumption (apart from some edge cases in LAMMPS supporting particle flow / GCMC simulations, which i-PI doesn't do AFAIK). Checking the number of atoms will be fast but note that we don't get identities from a code like i-PI (see the signature of the `calculate()` interface).

Reply:
(removing the assumption of fixed number of atoms is a pet peeve of mine, since I worked on GCMC simulations for my PhD)

I'm fine with having the assumption clearly documented for now!

Reply:
Ah, sorry, didn't mean to call GCMC an "edge case"! I'll add a check for # atoms consistency, since that's easy to do here.